pytorch_lightning/utilities/argparse.py

```diff
@@ -134,7 +135,7 @@ def get_init_arguments_and_types(cls) -> List[Tuple[str, Tuple, Any]]:
     return name_type_default
 
 
-def _get_abbrev_qualified_cls_name(cls):
+def _get_abbrev_qualified_cls_name(cls: Type['pl.Trainer']) -> str:
```

Comment:
same here, it would just be the type of a python class.

Reply:
@carmocca - Should this typing be also changed to just `Any`? Thanks :)

Reply:
Yes!

Reply:
Thanks, it's fixed now.